docs/design.md

```diff
@@ -30,6 +30,15 @@ parameters:
     type: int
     default: 3306
 ```
+Or with an empty parameters field
+```yaml
+name: fusor/etherpad-apb
+description: Note taking web application
+bindable: true
+async: optional
+parameters: []
```

Comment:
We could create a placeholder example, call it 
```yaml
name: example/no-parameters 
description: Example showing no parameters
bindable: false
async: optional
parameters: []
```

And keep an issue open to update once we create an apb that does take no parameters?